.gitignore

```diff
@@ -1,3 +1,4 @@
+.idea/
```

Comment:
This isn't project related and should be done in your local global `.gitignore` file.
https://stackoverflow.com/a/7335487/4907524